Portal/src/Datahub.Core/Model/Datahub/Datahub_Project.cs

```diff
@@ -315,6 +315,7 @@ public TerraformWorkspace ToResourceWorkspace(List<TerraformUser> users)
         {
             Name = Project_Name,
             Acronym = Project_Acronym_CD,
+            BudgetAmount = Convert.ToDouble(Project_Budget),
```

Comment:
any particular reason our Project_Budget is not a double in the first place?

Reply:
We have to change the DB, it has been there forever as a decimal.